### ISSUE TITLE
Translation activator activity

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#29317]
Translation activator activity is a direct child of translation regulator, nucleic acid binding,
whereas the repressor activity is a direct child of translation regulator

![image](https://github.com/user-attachments/assets/7ce8a2a9-0782-45fc-89b5-befc5b3409d8)

I wonder if something can be an activator without binding the RNA. For example cbp7/8 / sls1/mtf2/ppr4/mrh5 in the model below
PMID: 34634819

@ValWood 

![image](https://github.com/user-attachments/assets/07ea7999-15dd-4d01-b5f5-09b8cd9ed117)

Figure 12. Model of the cooperative interplay between ribosomal isoforms and translational activators to localize and coordinate synthesis and assembly
of OXPHOS subunits. Translational activators of cytb and cox1, Cbp7/8 and Mrh5/Ppr4/Mtf2/Sls1 respectively, could preferentially interact with either
Mug178 or Mrp51 ribosomes, the translated mRNA and the membrane, in order to localize Cytb and Cox1 synthesis into the sub-regions optimal for
subsequent assembly. Similarly, cox3 translation initiation would be favored on Mug178-containing SSU. Mrp51 ribosomes are proposed to synthesize
most of the other proteins, including a low amount of Cox3 protein, maybe through translation of the bi-cistronic cox1-cox3 mRNA. Small (SSU) and
large (LSU) ribosomal subunits have been indicated.

**Comment by ValWood** [comment]
Agreed. See https://github.com/geneontology/go-ontology/issues/29089
and add anything additional to this ticket.
v

**Comment by ValWood** [comment]
closing as duplicate https://github.com/geneontology/go-ontology/issues/29089, please add anything outstanding to that ticket
